sdcm/utils.py

```diff
@@ -177,15 +177,16 @@ def remove_comments(data):
 
 class S3Storage(object):
     @staticmethod
-    def generate_url(file_path):
-        job_name = get_job_name()
+    def generate_url(file_path, testid=None):
+        prefix_path = testid if testid else get_job_name()
+        # job_name = get_job_name()
```

Comment:
- remove support for job_name
- upload screenshot also to the test-id dir

Reply:
rename testid to prefix. left as parameter of method, the goal is not to import cluster to utils for one function.
screenshots uploaded to test_id dir.